src/components/Modal/Modal.jsx

```diff
@@ -46,6 +47,9 @@ class Modal extends React.Component {
     if (event.which === 27) {
       event.preventDefault();
       event.stopPropagation();
+      if (this.props.closeButton) {
+        this.props.closeButton.callback();
+      }
```

Comment:
hey @federicoweber I noticed that the modal would close but wouldn't execute the callback so the state wouldn't change within the app. I added this one for the closeButton but I wonder if you think it's worth adding all the action callbacks? I don't know if their callback will always be open/close?

Reply:
Good catch @gmzjuliana, yeah we can't really trigger those other two callbacks in here. There is no direct relation, with the action buttons, so it might create side effects.